collect_app/src/main/java/org/odk/collect/android/tasks/FormLoaderTask.java

```diff
@@ -258,8 +258,8 @@ private FormDef createFormDefFromCacheOrXml(String formPath, File formXml) {
                 Timber.i("Loaded in %.3f seconds. Now saving to cache.",
                         (System.currentTimeMillis() - start) / 1000F);
                 final long start2 = System.currentTimeMillis();
-                serializeFormDef(formDefFromXml, formPath);
-                Timber.i("Saved to cache in %.3f seconds.",
+                boolean saved = cacheFormDefIfNew(formDefFromXml, formPath);
+                Timber.i((saved ? "Saved" : "Found existing cached form") + " in %.3f seconds.",
```

Comment:
Timber offers vararg formatting, might want to swap this to:
`Timber.i( "%s in %.3f seconds.", saved ? "Saved" : "Found existing cached form");`
To avoid the string addition.

Reply:
Good to know for next time. Thanks.